### DISCUSSION TITLE
Timestamps preference

### DISCUSSION THREAD

**Discussion by aceofspaders** [AprilSylph/XKit-Rewritten#290] [Suggestion Box]
Hi!
Could you, by any chance, make a preference that can be switched on where timestamp will be shown only when a user hovers their cursor over a post, like it was back on NewXKit? 
I'm sorry if it's oddly specific but seeing timestamp all the time doesn't feel right to me - something about the way posts look with it - while I do enjoy being able to check when a certain post was made.
Thank you in advance for taking the time to read this,
Di

**Comment by AprilSylph** [comment]
Timestamps will be removed soon in favour of Tumblr's upcoming native timestamps feature.
